Home of Giganto throne treasure room crash

Flipping the lever back up causes a crash. Caused by patch to DRGN22/807/1

```
00:03:31.343 [Hardware legend.game.scripting.ScriptState:1236] ERROR: ScriptState[10] Submap object 0 (file 0) crashed!
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1237] ERROR: File Submap object 0 (DRGN2/807/1) CALL @ 0xa30
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1238] ERROR: Parameters:
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1239] ERROR:   Op param: 0x2a3
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1241] ERROR:   0: script[10].stor[24] 0x5
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1241] ERROR:   1: script[10].inl[0xa38] 0x0
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1244] ERROR: Storage:
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   0: 0xa
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   1: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   2: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   3: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   4: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   5: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   6: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   7: 0x10000
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   8: 0x6a9b5
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   9: 0xffffffef
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   10: 0x3e824
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   11: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   12: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   13: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   14: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   15: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   16: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   17: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   18: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   19: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   20: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   21: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   22: 0x0
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   23: 0x0
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   24: 0x5
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   25: 0x0
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   26: 0x0
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   27: 0x11
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   28: 0x3
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   29: 0x7e0
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   30: 0x0
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   31: 0x7b
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1246] ERROR:   32: 0xffffffff
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1249] ERROR: Registry IDs:
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1256] ERROR: Call stack:
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1258] ERROR:   0: Submap object 0 (DRGN2/807/1) 0xa30
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1258] ERROR:   1: Submap object 0 (DRGN2/807/1) 0x1fd8
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1258] ERROR:   2: Submap object 0 (DRGN2/807/1) 0x1210
00:03:31.344 [Hardware legend.game.scripting.ScriptState:1261] ERROR: Disassembly:
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1281] ERROR:   a08: call SMap::scriptLoadSobjAnimation, stor[24], 0x1d ; scriptIndex, animIndex
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1281] ERROR:   a14: call SMap::scriptToggleAnimationDisabled, stor[24], 0x0 ; scriptIndex, disabled
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1281] ERROR:   a20: return
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1281] ERROR:   LABEL_42:
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1281] ERROR:   a24: call SMap::scriptLoadSobjAnimation, stor[24], 0x1e ; scriptIndex, animIndex
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1284] ERROR:   a30: call SMap::scriptToggleAnimationDisabled, stor[24], 0x0 ; scriptIndex, disabled
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1285] ERROR:   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1288] ERROR:   LABEL_43:
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1288] ERROR:   a3c: return
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1288] ERROR:   
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1288] ERROR:   ; SUBROUTINE
00:03:31.383 [Hardware legend.game.scripting.ScriptState:1288] ERROR:   LABEL_44:
00:03:34.075 [Hardware legend.core.RenderEngine:996] ERROR: Shutting down due to exception:
java.lang.RuntimeException: An error occurred while ticking script 10
	at legend.game.scripting.ScriptManager.executeScriptFrame(ScriptManager.java:203)
	at legend.game.scripting.ScriptManager.tick(ScriptManager.java:61)
	at legend.game.Scus94491BpeSegment.lambda$gameLoop$46(Scus94491BpeSegment.java:414)
	at legend.core.RenderEngine.lambda$init$21(RenderEngine.java:607)
	at legend.core.opengl.Window$Events.onDraw(Window.java:746)
	at legend.core.opengl.Window.tickFrame(Window.java:386)
	at legend.core.opengl.Action.run(Action.java:46)
	at legend.core.opengl.Action.tick(Action.java:32)
	at legend.core.opengl.Window.run(Window.java:363)
	at legend.core.RenderEngine.run(RenderEngine.java:994)
	at legend.core.GameEngine.start(GameEngine.java:248)
	at legend.game.Main.main(Main.java:29)
Caused by: java.lang.NullPointerException: Cannot read field "innerStruct_00" because "legend.game.Scus94491BpeSegment_800b.scriptStatePtrArr_800bc1c0[legend.game.scripting.Param.get()]" is null
	at legend.game.submap.SMap.scriptToggleAnimationDisabled(SMap.java:1937)
	at legend.game.scripting.ScriptState.scriptExecuteSubFunc(ScriptState.java:1029)
	at legend.game.scripting.ScriptState.runOp(ScriptState.java:623)
	at legend.game.scripting.ScriptState.executeFrame(ScriptState.java:410)
	at legend.game.scripting.ScriptManager.executeScriptFrame(ScriptManager.java:199)
	... 11 more
```

--- a/patches/scripts/DRGN22/807/1.config.csv
+++ b/patches/scripts/DRGN22/807/1.config.csv
@@ -1,0 +1,1 @@
+table_length, 2224, 11

